napari/utils/misc.py

```diff
@@ -46,6 +46,11 @@ def is_iterable(arg, color=False):
         return True
 
 
+def is_complex(obj):
```

Comment:
maybe this should be named `guess_complex` (I think we may have used that convention elsewhere)